CONTRIBUTING.md

```diff
@@ -0,0 +1,13 @@
+# Contributing to the project
+
+## PR process
+* When you open a PR, add a few reviewers. If the PR solves a GitHub issue, make sure
+  that the user who opened the issue is one of the reviewers.
+* If you do not see any progress on the PR over a couple of days, feel free to put a
+  comment tagging one or two users asking them to take the time to review your change.
+* The reviewer will perform the code review. If they are satisfied with the change, and
+  they feel like the change does not require a second pair of eye, they will merge the
+  PR.
+* In the event that the PR is large enough to require 2 reviewers, the 1st reviewer will
```

Comment:
Rephrasing proposal: "If the PR is large it may be worth getting a second opinion. In that case the 1st reviewer will put a comment asking for another set of eyes. The second reviewer is responsible for merging it."

Reply:
Partially rephrased. We did not specify that the 2nd reviewer would be brought in once there was a first approval. Therefore there will be 2 reviewers commenting at the same time. I believe it's the responsibility of the last approving reviewer, not the last reviewers brought in.

Feel free to bring this topic during tomorrow's stand-up.

Reply:
At the end of the day I think that the reviewer, who put most energy into it should be the one getting the credit and merging the PR. I don't think that it can get deduced of the sequence of approval.